frontend/package.json

```diff
@@ -72,7 +72,7 @@
     "redux-thunk": "^2.4.1",
     "sass": "^1.56.2",
     "showdown": "^1.9.1",
-    "yaml": "^2.1.3"
+    "yaml": "^2.2.2"
```

Comment:
Can I ask why bumping the version of yaml in this PR?

Reply:
Looks like you didn't clean up the commit history so you are bringing some other commits into this PR.

Reply:
@DaoDaoNoCode looks like it's causing the issue when switching the target branch from `main` to `f/byon`

Reply:
yes, cause you based this branch on main rather than the feature branch, you need to either reset your branch onto `f/byon` or create a new PR.

Reply:
It should be good now.